save-demo/src/main/kotlin/com/saveourtool/save/demo/service/DemoService.kt

```diff
@@ -131,7 +154,6 @@ class DemoService(
      * @param projectName saveourtool project name
      * @return [Demo] connected with project [organizationName]/[projectName] or null if not present
      */
-    @Transactional
```

Comment:
or `@Transactional(readonly=true)`?

Reply:
Probably 😃

Reply:
The point is that Idea gives me an error:
```
@Transactional self-invocation (in effect, a method within the target object calling another method of the target object) does not lead to an actual transaction at runtime 
```
 on `findBySaveourtoolProject` invocation in method:
```Kotlin
fun findBySaveourtoolProjectOrNotFound(
    organizationName: String,
    projectName: String,
    lazyMessage: () -> String,
): Mono<Demo> = blockingToMono {
    findBySaveourtoolProject(organizationName, projectName)
}.switchIfEmptyToNotFound(lazyMessage)
```

Reply:
You cannot call a transactional method from another method in the same class

Reply:
Discussed offline - decided to ignore the warning 😁